certified-connectors/Webhood URL Scan/apiDefinition.swagger.json

```diff
@@ -0,0 +1,372 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Webhood - Beta API",
```

Comment:
Can you please remove word "API" as it is reserved word.  Please refer this link https://learn.microsoft.com/en-us/connectors/custom-connectors/certification-submission#give-your-connector-a-title